docs/user/pull-requests.rst

```diff
@@ -24,6 +24,14 @@ Build status report
 
        GitHub build status reporting
 
+Build overview with changed files
+    We create a comment on the pull request including a link to the preview of the documentation,
+    and a list of the files that changed in the pull request.
```

Comment:
I have a customer asking about what version were we comparing against and if they can change that. I think it's important to make this clear here.

Also, we don't the files changed in the PR, but between the rendered versions.

```suggestion
    We create a comment on the pull request including a link to the preview of the documentation,
    and a list of the files that changed between the base version (``latest`` by default) and the version from the pull request.
```